models/reviews.model.js

```diff
@@ -36,3 +36,34 @@ exports.updateReviewById = (id, inc_votes) => {
       }
     });
 };
+
+exports.selectReviews = (query) => {
+  const validCategories = [
+    "euro game",
+    "dexterity",
+    "social deduction",
+    "children's games",
+  ];
+
+  if (Object.keys(query).length === 0) {
+    return db
+      .query(
+        "SELECT reviews.*, COUNT (comment_id) AS comment_count from reviews left join comments on reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY created_at DESC"
+      )
+      .then(({ rows }) => {
+        return rows;
+      });
+  } else if (validCategories.includes(query.category)) {
+    return db
+      .query(
+        `SELECT reviews.*, COUNT (comment_id) AS comment_count from reviews left join comments on reviews.review_id = comments.review_id WHERE category = '${query.category}' GROUP BY reviews.review_id ORDER BY created_at DESC`
+      )
+      .then(({ rows }) => {
+        return rows;
+      });
+  } else if (query.category === undefined) {
+    return Promise.reject({ status: 400, message: "Query invalid" });
+  } else {
```

Comment:
this can be made a little more 'dry'  just using one SQL query that you can add to depending on if a category is passed in 
https://notes.northcoders.com/courses/js-back-end/complex-queries
have a look at the query string's in these notes and see if you could implement something along these lines here!